.github/workflows/ci.yml

```diff
@@ -14,14 +14,26 @@ jobs:
         ruby:
           - '2.6'
           - '2.7'
+          - '3.0'
+          - '3.1'
         gemfile:
           - rails4.2
           - rails5.1
           - rails5.2
           - rails6.0
           - rails6.1
+          - rails7.0
         exclude:
           - {ruby: '2.7', gemfile: rails4.2}
+          - {ruby: '2.6', gemfile: rails7.0}
+          - {ruby: '3.0', gemfile: rails4.2}
```

Comment:
Something is odd with the order of this list.

Reply:
Fixed!